docs/deployment.md

```diff
@@ -50,21 +50,45 @@ Bootstrap cdk, replacing `<profile name>`
 npx cdk bootstrap --profile <profile name>
 ```
 
-## Deploy the cdk stack
-
-Deploy the cdk stack, replacing `<profile name>` and env `<production or staging>`
+Then, run VPC configuration that cannot be done via cdk
 
 ```sh
-# Run VPC configuration that cannot be done via cdk
 npm run configureVpc
+```
+
+## Deploy the cdk stack
 
-npm run cdk:deploy --profile <profile name> --context env=<production or staging>
-# Add `--require-approval never` to skip the yes/no
+First, deploy the cdk stack with a http internal ALB, replacing `<profile name>` and env
```

Comment:
General suggestion is to spell out acronyms (VPC, ALB) the first time you use them and/or link to the docs for the AWS service. It's tedious and not totally necessary but potentially kind to a future maintainer/person trying to take inspiration from your work!